src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs

```diff
@@ -23,8 +23,8 @@ public class MemoryCache : IMemoryCache
         internal readonly ILogger _logger;
 
         private readonly MemoryCacheOptions _options;
-        private readonly ConcurrentDictionary<object, CacheEntry> _entries;
 
+        private ConcurrentDictionary<object, CacheEntry> _entries;
```

Comment:
volatile?

Reply:
I am not sure if it's needed. @stephentoub thoughts?

Reply:
Doesn't look like it's needed